recipe/meta.yaml

```
@@ -21,41 +21,139 @@ requirements:
+    - plotly.graph_objs.heatmap
+    - plotly.graph_objs.heatmapgl
+    - plotly.graph_objs.histogram
+    - plotly.graph_objs.histogram2d
+    - plotly.graph_objs.histogram2dcontour
+    - plotly.graph_objs.icicle
+    - plotly.graph_objs.image
+    - plotly.graph_objs.indicator
+    - plotly.graph_objs.isosurface
+    - plotly.graph_objs.layout
+    - plotly.graph_objs.mesh3d
+    - plotly.graph_objs.ohlc
+    - plotly.graph_objs.parcats
+    - plotly.graph_objs.parcoords
+    - plotly.graph_objs.pie
+    - plotly.graph_objs.pointcloud
+    - plotly.graph_objs.sankey
+    - plotly.graph_objs.scatter
+    - plotly.graph_objs.scatter3d
+    - plotly.graph_objs.scattercarpet
+    - plotly.graph_objs.scattergeo
+    - plotly.graph_objs.scattergl
+    - plotly.graph_objs.scattermapbox
+    - plotly.graph_objs.scatterpolar
+    - plotly.graph_objs.scatterpolargl
+    - plotly.graph_objs.scatterternary
+    - plotly.graph_objs.splom
+    - plotly.graph_objs.streamtube
+    - plotly.graph_objs.sunburst
+    - plotly.graph_objs.surface
+    - plotly.graph_objs.table
+    - plotly.graph_objs.treemap
+    - plotly.graph_objs.violin
+    - plotly.graph_objs.volume
+    - plotly.graph_objs.waterfall
+    - plotly.validators
+    - plotly.validators.bar
+    - plotly.validators.barpolar
+    - plotly.validators.box
+    - plotly.validators.candlestick
+    - plotly.validators.carpet
+    - plotly.validators.choropleth
+    - plotly.validators.choroplethmapbox
+    - plotly.validators.cone
+    - plotly.validators.contour
+    - plotly.validators.contourcarpet
+    - plotly.validators.densitymapbox
+    - plotly.validators.frame
+    - plotly.validators.funnel
+    - plotly.validators.funnelarea
+    - plotly.validators.heatmap
+    - plotly.validators.heatmapgl
+    - plotly.validators.histogram
+    - plotly.validators.histogram2d
+    - plotly.validators.histogram2dcontour
+    - plotly.validators.icicle
+    - plotly.validators.image
+    - plotly.validators.indicator
+    - plotly.validators.isosurface
+    - plotly.validators.layout
+    - plotly.validators.mesh3d
+    - plotly.validators.ohlc
+    - plotly.validators.parcats
+    - plotly.validators.parcoords
+    - plotly.validators.pie
+    - plotly.validators.pointcloud
+    - plotly.validators.sankey
+    - plotly.validators.scatter
+    - plotly.validators.scatter3d
+    - plotly.validators.scattercarpet
+    - plotly.validators.scattergeo
+    - plotly.validators.scattergl
+    - plotly.validators.scattermapbox
+    - plotly.validators.scatterpolar
+    - plotly.validators.scatterpolargl
+    - plotly.validators.scatterternary
+    - plotly.validators.splom
+    - plotly.validators.streamtube
+    - plotly.validators.sunburst
+    - plotly.validators.surface
+    - plotly.validators.table
+    - plotly.validators.treemap
+    - plotly.validators.violin
+    - plotly.validators.volume
+    - plotly.validators.waterfall
+  requires:
+    - pip
+    - {{ pin_compatible("numpy") }}
```

```suggestion
    - numpy
```
There is no need for pin_compatible here.
See when to use pin_compatible: https://docs.conda.io/projects/conda-build/en/stable/resources/variants.html#pinning-at-the-recipe-level

---

good catch! thank you!